.travis.yml

```diff
@@ -1,10 +1,19 @@
 language: go
 go_import_path: github.com/SUSE/saptune
 go:
-  - 1.7.x
+  - 1.10.x
```

Comment:
@angelabriel Can this upgrade cause problems on old releases like SLE-12-SP2?

Reply:
There is no problem with the backward compatibility of the saptune binary. Even if built with 1.10, it will run without problems on a system with go version 1.7 or lower. And version 1.10 of the go package is already available for SLE12SP2 too.
But the new compiler version is more 'strict' in some checks and so our build in OBS can fail during the check/test step (see https://bugzilla.suse.com/show_bug.cgi?id=1120741).
So it makes sense to use the newer version of go in our travis tests, to see the problems a little bit earlier.

Reply:
@angelabriel OK, good to know that. Thanks.